hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java

```diff
@@ -727,4 +728,17 @@ void testTopicSubmitMessageResultCreation() {
         Assertions.assertThrows(NullPointerException.class, () -> new TopicSubmitMessageResult(null, validStatus));
         Assertions.assertThrows(NullPointerException.class, () -> new TopicSubmitMessageResult(validTransactionId, null));
     }
+    
+    @Test
+    void testTopicDeleteResult() {
```

Comment:
@rdodiya looks good but change `testTopicDeleteResult()` method name to `testTopicDeleteResultCreation()` for consistence in the naming convetion.